src/pubd/repository.rs

```diff
@@ -906,14 +889,6 @@ impl RrdpServer {
         keep
     }
 
-    // Find where to truncate old notification files
```

Comment:
Shouldn't this PR also remove the no longer used config settings, e.g. `retention_old_notification_files_seconds` ?

Reply:
yes.. now removed